fastchat/conversation.py

```diff
@@ -130,6 +130,8 @@ def get_prompt(self) -> str:
             for i, (role, message) in enumerate(self.messages):
                 tag = self.roles[i % 2]
                 if message:
+                    if type(message) is tuple:
```

Comment:
Update the docstring of `message` to specify the message format for images
https://github.com/lm-sys/FastChat/blob/77932a1eb3dbb385fbf4530f0edc76f1c8c621bc/fastchat/conversation.py#L45-L46